go.mod

```diff
@@ -1,3 +1,108 @@
 module github.com/medibloc/panacea-doracle
 
 go 1.17
+
+require (
+	github.com/cosmos/cosmos-sdk v0.45.4
```

Comment:
Wouldn't we need panacea-core?
Umm, Is it because our fix hasn't been applied to the master branch yet?

Reply:
Ja. When you import the panacea-core, please refer to https://medibloc.gitbook.io/panacea-core/guide/installation#import-panacea-core-as-a-go-dependency for specifying `replace` directives (Please double-check package versions in the `replace` directives).

Reply:
Thank you for your comments. When I import panacea-core, I'll check it!